packages/atomic/cypress/integration/facet-date.cypress.ts

```diff
@@ -0,0 +1,233 @@
+import {setUpPage, shouldRenderErrorComponent} from '../utils/setupComponent';
```

Comment:
Why not name the file `date-facet.cypress.ts`?

Reply:
well at the beginning I named it as `facet ...` to keep all the files closed to each other, however, like recommendation from @ThibodeauJF , I already renamed, and put all of them into a 'facet' folder,